backend/src/main/java/gov/cdc/usds/simplereport/db/model/TestOrder.java

```diff
@@ -76,20 +52,16 @@ public PatientAnswers getAskOnEntrySurvey() {
 		return askOnEntrySurvey;
 	}
 
-	public TestResult getTestResult() {
-		return result;
-	}
-
 	public LocalDate getDateTested() {
```

Comment:
This could probably be removed entirely now and replaced with testEvent.createdAt, right?

Reply:
Makes sense to me.